docs/integration/assets_categories/device/sentinelone_edr.md

```diff
@@ -0,0 +1,66 @@
+uuid: f47ac10b-58cc-4372-a567-0e02b2c3d479
+name: SentinelOne EDR
+type: asset
+
+## Overview
+
+SentinelOne is an Endpoint Detection and Response (EDR) solution that provides comprehensive endpoint protection and monitoring. This setup guide shows how to forward device assets monitored by SentinelOne EDR to Sekoia.io for centralized asset management and security operations.
+
+- **Vendor**: SentinelOne
+- **Product**: SentinelOne EDR
+- **Supported environment**: Cloud
```

Comment:
I detected that some other asset connectors documentation, we are talking about `SaaS`.
I think we have to refine the documentations for consistency